src/main/java/com/example/bulletinboardapp/controller/UserController.java

```diff
@@ -0,0 +1,40 @@
+package com.example.bulletinboardapp.controller;
+
+import com.example.bulletinboardapp.dto.UpdateUser;
+import com.example.bulletinboardapp.service.impl.UserServiceImpl;
+import io.swagger.annotations.Api;
+import com.example.bulletinboardapp.dto.NewPassword;
+import com.example.bulletinboardapp.entity.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@Api(tags = "Пользователи")
+@RestController
+@RequestMapping("/users")
+@CrossOrigin(value = "http://localhost:3000")
+@RequiredArgsConstructor
+public class UserController {
+
+    private final UserServiceImpl userService;
+
+    @PostMapping("/setPassword")
+    public NewPassword setPassword (@RequestBody NewPassword newPassword){
+        return newPassword;
+    }
+
+    @GetMapping("/me")
+    public User getUser(){
+        return new User();
+    }
+
+    @PatchMapping("/me")
+    public UpdateUser updateUser(@RequestBody UpdateUser updateUser){
+        return new UpdateUser();
+    }
+
+    @PatchMapping("/me/image")
```

Comment:
```suggestion
    @PatchMapping(value = "/me/image", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```
вот так должно быть, иначе работать не будет